dataframe/src/main/groovy/com/elintegro/erf/widget/vue/SnackbarWidgetVue.groovy

```diff
@@ -53,26 +55,35 @@ class SnackbarWidgetVue extends WidgetVue{
         String y = field?.y?:"top"
         String x = field?.x?:"right"
         def timeout = field?.timeout?:6000
-        dataframe.getVueJsBuilder().addToComputedScript("""
+   /*     dataframe.getVueJsBuilder().addToComputedScript("""
               alertProp(){
                return this.\$store.state.${dataframe.dataframeName}.alertProp;
            },\n
-        """)
+        """)*/
         VueStore store = dataframe.getVueJsBuilder().getVueStore()
-        store.addToState("""
+        dataframe.getVueJsBuilder().addToDataScript("""
              alertProp:{
                 snackbar: false,
                 alert_type: '',
                 alert_message: ''
             },\n
         """)
-        store.addToMutation("""
+        dataframe.getVueJsBuilder().addToMethodScript("""
+         showMessage(responseData){
```

Comment:
See if this javascript is generated for each Dataframe with the snackbar (Not good!!!), so we can create this function in erfCommon.js and call it with exon.showMessage(...)